previousnext/agov
# 28684 module whitelist

Steps to test:
* As administer goto admin/config/system/module-whitelist and set which modules are whitelisted
* Create a user with 'Site Editor' role
* As 'Site Editor' goto admin/modules only the whitelisted modules should show
